crates/it-json-serde/src/errors.rs

```diff
@@ -14,8 +14,13 @@
  * limitations under the License.
  */
 
-mod ivalues_to_json;
-mod json_to_ivalues;
+use thiserror::Error;
 
-pub(crate) use ivalues_to_json::ivalues_to_json;
-pub(crate) use json_to_ivalues::json_to_ivalues;
+#[derive(Debug, Error)]
+pub enum ItJsonSerdeError {
+    #[error("cannot serialize IValues to Json: {0}")]
+    SerializationError(String),
```

Comment:
Maybe rename these variants to just `Se` and `De`, in this case it would be accessible by `ItJsonSerdeError::Se`, `ItJsonSerdeError::De`

Reply:
And in the ideal situation it shouldn't be a `String`, but another `enum` with more strict errors. But prob it's better to that in another PR, I suggest just to leave TODO here in this PR.

Reply:
Yes, `De` and `Se` are fine, and strict errors can be done later, when we will be paying tech dept